### PR TITLE
Homework 5 Week 5

### PR DESCRIPTION
**Pull request by jflinkstrom**

### CODE REVIEW

**Comment by jflinkstrom** [comment]
I commented out the test classes that connect to a DB so the mvn test can work (per what was said in the chat)... hopefully this works.

File: src/main/java/com/origamisoftware/teach/advanced/apps/stockquote/BasicStockQuoteApplication.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import java.util.List;
+
+/**
+ * A simple application that shows the StockService in action.
+ */
+public class BasicStockQuoteApplication {
+
+    private StockService stockService;
+
+    // an example of how to use enum - not part of assignment 3 but useful for assignment 4
```
Comment (smarks) [inline src/main/java/com/origamisoftware/teach/advanced/apps/stockquote/BasicStockQuoteApplication.java:19]: remove cruft like this

File: src/main/java/com/origamisoftware/teach/advanced/apps/stockquote/BasicStockQuoteApplication.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+     */
+    private static void exit(ProgramTerminationStatusEnum statusCode, String diagnosticMessage) {
+        if (statusCode == ProgramTerminationStatusEnum.NORMAL) {
+            System.out.println(diagnosticMessage);
+        } else if (statusCode == ProgramTerminationStatusEnum.ABNORMAL) {
+            System.err.println(diagnosticMessage);
+        } else {
+            throw new IllegalStateException("Unknown ProgramTerminationStatusEnum.");
+        }
+        //System.exit(statusCode.getStatusCode()); // can't run "mvn test" with a system.exit() call... haven't figured out a work around
```
Comment (smarks) [inline src/main/java/com/origamisoftware/teach/advanced/apps/stockquote/BasicStockQuoteApplication.java:106]: don't include commented out remove it instead. (the history will remain source code control)

**Review by smarks — APPROVED** [review]
Basically, a good job here. I made a few comments.

**Comment by jflinkstrom** [comment]
merge hw5 to main

**Pull request merged**